test/helpers.py

```diff
@@ -64,11 +70,12 @@ def _compile(code):
     os.chdir(os.path.dirname(tex_file))
 
     # compile the output to pdf
+    cmds = dict(
+        latex=["pdflatex", "--interaction=nonstopmode"],
+        context=["context", "--nonstopmode"],
+    )
     try:
-        subprocess.check_output(
-            ["pdflatex", "--interaction=nonstopmode", tex_file],
-            stderr=subprocess.STDOUT,
-        )
+        subprocess.check_output(cmds[flavor] + [tex_file], stderr=subprocess.STDOUT)
     except subprocess.CalledProcessError as e:
         print("pdflatex output:")
```

Comment:
You probably want to adapt this string too.

Reply:
will do!